grails-app/services/au/org/ala/biocache/hubs/MessageSourceCacheService.groovy

```diff
@@ -2,6 +2,8 @@ package au.org.ala.biocache.hubs
 
 import grails.plugin.cache.Cacheable
 
+import elurikkus.commons.ExtendedPluginAwareResourceBundleMessageSource
```

Comment:
Something missing here?

Reply:
I deleted the class from occurrences previously, but forgot to change the imports